Gemfile.lock

```diff
@@ -155,7 +155,7 @@ GEM
     crass (1.0.6)
     data-confirm-modal (1.6.3)
       railties (>= 3.0)
-    debug_inspector (1.0.0)
+    debug_inspector (1.1.0)
```

Comment:
### [debug_inspector](https://github.com/banister/debug_inspector)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.0.0 → 1.1.0

[_[change-log](https://github.com/banister/debug_inspector/releases), [source-code](https://github.com/banister/debug_inspector)_]





<details>
<summary>Commits</summary>

A change of **13** commits. See the full changes on [the compare page](https://github.com/banister/debug_inspector/compare/v1.0.0...v1.1.0).

These are the first 10 commits:
- (7bb0951) [Add gem description and changelog URI](https://github.com/banister/debug_inspector/commit/7bb09517282a41b3fdf3c204f46acebf17896b51)
- (0587601) [Load extension from the ext directory](https://github.com/banister/debug_inspector/commit/058760187fbdc7a7ca78c90b1c6251b653379a8b)
- (5132abe) [List the contents of the installed gem in CI](https://github.com/banister/debug_inspector/commit/5132abed01947c29bccdef79f858433478c604de)
- (76b4060) [Try loading from both locations](https://github.com/banister/debug_inspector/commit/76b4060428dcf24a3b426c3253e96966508c5ab1)
- (8029a5a) [CI install without rake task](https://github.com/banister/debug_inspector/commit/8029a5a5ee54e211128d655555fd4403b29ca48c)
- (bb70e4a) [Ruby 3.0 on Windows CI](https://github.com/banister/debug_inspector/commit/bb70e4a5b7302cfc81192545fe2b9bdda8ce3fef)
- (bc982f8) [Try including lib_dir when building](https://github.com/banister/debug_inspector/commit/bc982f8ded1802c3254b9c89f94bce35651c9098)
- (2680ba0) [Another attempt to get CI to build the gem](https://github.com/banister/debug_inspector/commit/2680ba043f39fdc2ec5a5938685409ce16552332)
- (1b1d576) [Merge pull request #30 from BetterErrors/fix/load-extension-…](https://github.com/banister/debug_inspector/commit/1b1d5761a46b7605eb456ce7edea76f798118fd5)
- (af72646) [Version 1.0.1rc1](https://github.com/banister/debug_inspector/commit/af72646b6bbb9e32eb5f82ad499d32a9dedad9f2)


</details>